tutorials/tutorial1_rivetedassembly.py

```diff
@@ -369,12 +389,15 @@ def define_grid(self, contour: vm.wires.Contour2D, number_rivet1: int, number_ri
                 grids.append(vm.Point2D(xmin + ratio1 * (n1 + 1), ymin + ratio2 * (n2 + 1)))
         return grids
 
-    def generate(self) -> List[PanelAssembly]:
+    def generate(self, progress_callback=lambda x: 0) -> List[PanelAssembly]:
```

Comment:
progress_callback input might interfere with usage in platform ?

Reply:
I have the same in other __init__ and it works in the platform, whith the workflow of this exemple too, I can remove it if you want !